pims/norpix_reader.py

```diff
@@ -150,12 +183,17 @@ def _verify_frame_no(self, i):
         if i >= self._image_count or i < 0:
             raise ValueError("Frame number is out of range: " + str(i))
 
+    def set_process_func(self, process_func):
```

Comment:
Yes, sorry my original comment lacked that context. Basically, in the next major revision of pims, all the `process_func` stuff will be gone. Instead one can/will use something like

```python
@pims.pipeline
def read_as_color(image):
    ...
    return color_image

if is_color(seq_file_opened_as_raw):
    seq = read_as_color(seq_file_opened_as_raw)
elif is_16bit(seq_file_opened_as_raw):
    ...
```

to make a "virtual" sequence object with the desired interpretation.

`pipeline` is already working. So if `set_process_func` is in this PR, I bet that within a few months it will be torn out. But my comment is more of an advisory than a requirement for this PR; someone else can make that change as we update all the other reader classes.